src/webapp/src/characters/Characters.css

```diff
@@ -0,0 +1,29 @@
+.characters-grid {
+  display: grid;
+  grid-template-columns: repeat(auto-fill, minmax(300px, 1fr));
+  grid-auto-rows: minmax(250px, auto);
+}
+.card {
+  background: black;
+  border: 3px solid rgba(255, 255, 255, 1);
+  border-radius: 30px;
+  color: white;
+  box-shadow: 0px 0px 35px 1px rgba(255, 255, 255, 0.8);
+  margin: 3rem;
+  overflow: hidden;
+  position: relative;
+  max-width: 300px;
+}
+.picture-wrap img {
+  height: 100%;
+  object-fit: cover;
+  width: 100%;
+}
+
+.card-footer {
+  padding: 1.5rem;
+}
+
+.character-name {
+  margin: 0;
+}
```

Comment:
Regarding this point I have a question I cannot answer without your help I believe.

We discussed and agreed to use `styled-components` yes, but I thought we were going to use those components through `storybooks` and for now we cannot. So in order to use `stlyed-components` I have to install them as a dependency in the webapp.

Because of the above, I thought I would create just some dummy css. Nevertheless, if this was a bad decision I though of two ways of solving it.

1. I could learn how to use and build the components with `storybooks` and then use them in our webbapp
2. I could install `styled-components` in the `webapp` and learn how to use them for styling in this particular case.

What do you think it is better in cases like this one?

Reply:
We gonna have styled components in storybook to style the simplest of the components. But also we gonna use them in the `webapp`. So both 1 and 2 are right in this case.

Reply:
Perfect, I will install `styled-components` in the `webapp` and I will change the Characters.css to practice because I haven't used it before. Now I have to go to the dentist to get a root canal! I am bit afraid.

Reply:
Hahha good luck!